src/main/java/cn/nukkit/block/BlockRedstoneWire.java

```diff
@@ -41,7 +41,7 @@ public int getId() {
 
     @Override
     public boolean place(Item item, Block block, Block target, BlockFace face, double fx, double fy, double fz, Player player) {
-        if (face != BlockFace.UP || !canBePlacedOn(target)) {
+        if (!canBePlacedOn(block.down())) {
```

Comment:
Maybe use `getSideVec(BlockFace.DOWN)` instead

Reply:
`canBePlacedOn(Vector3)` uses getBlock anyway so we can get the correct block here and directly use `canBePlacedOn(Block)` instead

Reply:
Ah sure 😄